README.md

```diff
@@ -31,13 +31,27 @@ targets: '[{"project": "Backlog", "section": "Development Done"}, {"project": "C
 if you don't want to move task omit `targets`.
 
 
-## Example usage
+## Examples
+
+
+#### Without special chracters:
 
 ```yaml
-uses: https://github.com/insurify/github-actions@v2.0.0
+uses: insurify/github-asana-action@v3.0.0
```

Comment:
I don't think you need to bump your major version. The change you're making is minor, and it's also backward compatible. All the existing patterns will continue working.

Reply:
If anyone wants to try the trigger phrase without asterisk in the description we won't be able to catch it as we have hardcoded the asterisk and a space in the previous versions. 
Though we can release a minor version of this release, thoughts ?

Reply:
sure, minor release is fine.